pages/main.js

```diff
@@ -0,0 +1,45 @@
+document.querySelector(".login_form").addEventListener("submit", generateTable);
+
+    function generateTable(event) {
+     event.preventDefault();
+
+     console.log(12123);
+
+     const firstName = document.getElementById("first-name").value;
+     const lastName = document.getElementById("last-name").value;
+     const birthYear = document.getElementById("birth-year").value;
+     const sex = document.querySelectorAll('input[name="sex"]:checked').value;
+    
+     const city = document.getElementById("city").value;
+     const address = document.getElementById("address").value;
+
+     const languages = [];
+
+     console.log([firstName, lastName, city, sex, birthYear, address, languages]);
+
+     const languageCheckboxes = Array.from(document.getElementsByName("language"));
```

Comment:
```
const languageCheckboxes = document.querySelectorAll('input[name="language"]:checked');
```

1. You get an array of checked values
2. In `for (var i = 0; i < languageCheckboxes.length; i++)` you don't have unnecessary iterations
3. You don't need `if (languageCheckboxes[i].checked)`
// ...